README.md

```diff
@@ -76,7 +76,29 @@ You can install the extension directly from the source. Follow the following ste
       - Navigate to _about://extensions_.
       - Click on "Load Unpacked" button (make sure you have enabled the _Developer mode_).
       - From the file explorer, choose `ccsearch-browser-extension/dist/opera`.
-
+## Debugging in Development Mode
+   - **Mozilla Firefox**
+      - Navigate to _about:debugging_ and from sidebar select _This Firefox_.
+      - Click on "Load Temporary Add-on" button. ![Load Temporary Add-on](debug-image/debuging-in-firefox.png)
+      - From the file explorer, choose `ccsearch-browser-extension/dist/firefox/manifest.json`.
+      - You will be now able to see CC search browser extension. Now click on the Inspect button. ![Inspect](debug-image/inspect-firefox.png)
+      - A new tab will open with firefox debugger. Now first, click on the _three dots_ and select "Disable Popup Auto-Hide". ![Disable Popup Auto-hide Image](debug-image/three-dots-firefox.png)
+      - Now click on the browser extension from top right corner and the DOM will be loaded with the content to debug. ![Firefox debug](debug-image/click-on-extension.png)
+   - **Google Chrome**
+      - Navigate to _chrome://extensions_.
+      - Make sure Developer mode in ON, if not, please ON it.
+      - Click on "Load Unpacked" button. ![Load Temporary Add-on](debug-image/chrome-load-unpacked.png)
+      - From the file explorer, choose `ccsearch-browser-extension/dist/chrome`.
+      - You will be now able to see CC search browser extension. Right click on the extension area and select "Inspect". ![Inspect in Chrome](debug-image/inspect-in-chrome.png)
+      - Developer tool will now open loaded with the content to debug. ![Debugging in Chrome](debug-image/debug-chrome.png)
```

Comment:
These screenshots are not correct. The DOM screenshot pasted is of `chrome://extensions` page and not of the extension itself.

Reply:
I will add a new screenshot. Thank you.

Reply:
@techievivek Please use the image that is actually of the chrome dev tools. The default layout of it is that it's not of full size and opened in a new window. This makes it easier to debug. I suggest you do the same for Opera as the developer will never have the debugger maximized as they will not be able to select the elements on the extension. I have taken some quick screenshots to give you the idea of what might work.

![Screenshot from 2020-02-23 19-56-01](https://user-images.githubusercontent.com/34679965/75113807-e0733e00-5676-11ea-97da-3018790835d0.png)
![Screenshot from 2020-02-23 19-56-58](https://user-images.githubusercontent.com/34679965/75113812-f1bc4a80-5676-11ea-87e4-6904d787b1db.png)

It would be good if you disable other extensions on chrome and opera while taking the screenshots.